commitizen/changelog.py

```diff
@@ -281,3 +281,71 @@ def incremental_build(new_content: str, lines: List, metadata: Dict) -> List:
     if not isinstance(latest_version_position, int):
         output_lines.append(new_content)
     return output_lines
+
+
+def get_smart_tag_range(
+    tags: List[GitTag], start: str, end: Optional[str] = None
+) -> List[GitTag]:
+    """Smart because it finds the N+1 tag.
+
+    This is because we need to find until the next tag
```

Comment:
May I know why do we need to find the next tag? For the case that `end` does not exist?

Reply:
Because we expect to receive the tag and 1 more in order to create the range of commits